### PR TITLE
feature/LPFG-258 - Docker build

### PR DESCRIPTION
**Pull request by superafroman**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ENV SPRING_PROFILES_ACTIVE production
+
+EXPOSE 8080
+EXPOSE 8081
+
+CMD java -jar /data/app.jar
+
+ADD build/libs/identity-service.jar /data/app.jar
+
+ADD https://github.com/Civil-Service-Human-Resources/lpg-terraform-paas/releases/download/hammer-0.1/hammer /bin/hammer
```
Comment (elxsy) [inline Dockerfile]: INFO - Do we need to add or pass ENV variables?

File: environ/ci [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
 ./gradlew build
-./gradlew test
\ No newline at end of file
+./gradlew test
+
+docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
+docker pull cshr/${SERVICE_NAME} || true
+
+docker build -t cshr/${SERVICE_NAME}:latest .
+
```
Comment (elxsy) [inline environ/ci]: INFO - Our new versioning (till get get the CI sorted
```
TAG =${TRAVIS_TAG:-$TRAVIS_COMMIT}
docker tag cshr/${SERVICE_NAME}:latest cshr/${SERVICE_NAME}:${TAG}
```

File: Dockerfile (hunk 1 above)
Comment (superafroman) [reply Dockerfile]: We can pass any env vars - they don't need to be defined in the Dockerfile. Normally I would define them if we need/want to specify a default for the container, like with the spring profile. But mostly our defaults will come from the application.properties in the code.

File: environ/ci (hunk 2 above)
Comment (superafroman) [reply environ/ci]: 👍 sorted

**Pull request merged**